edit/edit.css

```diff
@@ -24,6 +24,10 @@ body {
   margin-top: 1rem;
   margin-left: 1.7rem;
 }
+section {
+  box-sizing: border-box;
+  width: 100%;
```

Comment:
Why is this required? It breaks the layout in narrow width media queries. Disabling this rule doesn't change anything.